installer/bundle/bundle_skel.sh

```diff
@@ -315,6 +315,18 @@ compare_arch()
     esac
 }
 
+compare_install_type()
+{   
+    # If the bundle has an INSTALL_TYPE, check if the bundle being installed 
+    # matches the installer on the machine (rpm vs.dpkg)
+    if [ ! -z "$INSTALL_TYPE" ]; then
+        if [ $INSTALLER != $INSTALL_TYPE ]; then
+           echo "This kit is intended for ${INSTALL_TYPE} systems and cannot install on ${INSTALLER} system"
```

Comment:
Get rid of extra space after `${INSTALLER}` and change `system"` to `systems"` for consistency with other wording.